csl-variables.rnc

```diff
@@ -53,6 +54,30 @@ div {
     | "page-first"
     | "volume"
   
+  ## Title variables
+  variables.titles =
+    "collection-title"
+    | "collection-title-main"
```

Comment:
The main and sub versions don't need to be here. They could just be supplied as part of the data. Similar to how we don't have `issued-year`, `issued-month`. If someone wants to render just part of the title, they should use `cs:title`.

Reply:
So I move them back to the regular variables? Also the short versions stay with the string variables?